<?xml version="1.0"?>
<launch>

    <group ns="/ally2">

        <!-- Set ROS Params -->
        <param name="robot_estimator_on" type="bool" value="false" />
        <param name="go_rogue" type="bool" value="false" />

        <!-- Start the robot up -->
        <include file="$(find playground)/launch/robot.launch">
            <arg name="me" value="ally2" />
            <arg name="ally" value="ally1" />
        </include>

    </group>
</launch>
